src/Api/ResourceApiController.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Saritasa\LaravelControllers\Api;
+
+use Dingo\Api\Http\Request;
+use Dingo\Api\Http\Response;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Validation\ValidationException;
+use Saritasa\DingoApi\Traits\PaginatedOutput;
+use Saritasa\Enums\PagingType;
+use Saritasa\Exceptions\InvalidEnumValueException;
+use Saritasa\LaravelEntityServices\Contracts\IEntityService;
+use Saritasa\LaravelEntityServices\Contracts\IEntityServiceFactory;
+use Saritasa\LaravelEntityServices\Exceptions\EntityServiceException;
+use Saritasa\LaravelEntityServices\Exceptions\EntityServiceOperationException;
+use Saritasa\LaravelRepositories\DTO\SortOptions;
+use Saritasa\Transformers\IDataTransformer;
+
+/**
+ * Default controller to handle CRUD operation with models.
+ */
+class ResourceApiController extends BaseApiController
+{
+    use PaginatedOutput;
+
+    /**
+     * Entities services factory.
+     *
+     * @var IEntityServiceFactory
+     */
+    protected $entityServiceFactory;
+
+    /**
+     * Entity service to work with serve by this controller model.
+     *
+     * @var IEntityService
+     */
+    protected $entityService;
+
+    /**
+     * Serve model class.
```

Comment:
1) *Serve**d***
2) IMHO *handled* or *managed* are more appropriate. Translate of this words is the same, but IMHO it more appropriate for IT (possible we can discuss it in our group in Skype)